napari/layers/utils/plane_manager.py

```diff
@@ -77,3 +73,59 @@ def from_points(cls, a, b, c):
         plane_normal = np.cross(ab, ac)
         plane_position = np.mean(abc, axis=0)
         return cls(position=plane_position, normal=plane_normal)
+
+
+class PlaneList(EventedList):
```

Comment:
might be worth making this a `SelectableEventedList`, to simplify dealing with 'which plane am I dealing with', what do you think?

Reply:
That's a good idea, would make interactivity easier!